app/components/forms/admin/settings/images-form.js

```diff
@@ -0,0 +1,176 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+const { Component } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        large_width: {
+          identifier : 'large_width',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter width')
+            }
+          ]
+        },
+        large_height: {
+          identifier : 'large_height',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter height')
+            }
+          ]
+        },
+        large_quality: {
+          identifier : 'large_quality',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter quality')
+            }
+          ]
+        },
+        thumb_width: {
+          identifier : 'thumb_width',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter width')
+            }
+          ]
+        },
+        thumb_height: {
+          identifier : 'thumb_height',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter height')
+            }
+          ]
+        },
+        thumb_quality: {
+          identifier : 'thumb_quality',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter quality')
+            }
+          ]
+        },
+        event_icon_width: {
```

Comment:
@anu-007 No where across the project are js Variables in camel, case rather snake case is followed for all js files, this is fine.